test/utils/command.spec.js

```diff
@@ -486,16 +486,23 @@ describe('parseCommandLine', () => {
 
     test('short syntax', () => {
       const commandLine =
-        'job execute /Public/job -t targetName -w -o ./output.json -l ./log.json'
+        'job execute /Public/job -t targetName -w -o ./output.json -l ./log.json -s ./status.txt'
```

Comment:
the log should not arrive in JSON format (not sure if that fix is merged yet though)